python/pyspark/sql/connect/_typing.py

```diff
@@ -26,8 +26,12 @@
 import datetime
 import decimal
 
+from numpy import ndarray as NDArray
+
 from pyspark.sql.connect.column import Column
 
+ArrayLike = NDArray
```

Comment:
hm, should we do this alias?

Reply:
just copy it from pyspark

Reply:
let me remove it, since it's now used only once